proto/zetachain/zetacore/observer/confirmation_params.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package zetachain.zetacore.observer;
+
+option go_package = "github.com/zeta-chain/node/x/observer/types";
+
+message ConfirmationParams {
```

Comment:
It might be a good idea to add comments to explain what fast and slow are used for.

Reply:
added description for each field https://github.com/zeta-chain/node/pull/3461/commits/ea3f43b2cf2fc46c60459e7487ce0ddc60000194